### ISSUE TITLE
Decide on basemap to use for all three maps

### ISSUE THREAD

**Issue by ect123** [polyrhythm-project/polyrhythm-maps#26]
A base map should be used as a consistent backdrop for the data. There are several Leaflet base map providers, but some include a payment or API requirements. This list will give you some hints (inside the JS code) for ones that are free or not: [https://leaflet-extras.github.io/leaflet-providers/preview/](https://leaflet-extras.github.io/leaflet-providers/preview/).

Please decide on one that might best fit this project and add the information below. Once one is decided on, we can swap them in for each map.

- [ ] Corpus map
- [ ] Composers map
- [ ] Timeline map

**Comment by ect123** [comment]
Additionally, if a very light base map is picked, then popups might need to be styled with a drop shadow, background color, or border to set them apart from the base map.

**Comment by epoudrier** [comment]
I have identified four potential Esri maps. For the timeline map, it would be nice to have street names, so the best may be no. 1. For the corpus map, less detail is needed, so no. 2 might work. For the composer map, one of the two in no. 3 might work, but it depends on how we represent birth and death place and connect them.

1. Esri.WorldStreetMap 
(street names, but no clear country lines)

2. Esri.WorldTopoMap
(no street names beyond highways; no clear country lines)

3. Esri.NatGeoWorldMap or Esri.DeLorme (no street names beyond highways; country lines and capitals highlighted)

There were no Stamen maps that seemed preferable.

**Comment by egrguric** [comment]
@craigsapp will add the basemaps to the code based on documentation that Lily and Xilin have written. Documentation for adding new basemap is in Readme for Timeline map under "Solutions Documentation" section here: https://github.com/polyrhythm-project/polyrhythm-maps/tree/master/timeline.

**Comment by epoudrier** [comment]
Based on our discussion, it seems that the best option for the basemap is Esri.WorldStreetMap.

**Comment by egrguric** [comment]
Corpus and Timeline maps can both use Esri.WorldStreetMap; Composers map should use Esri.WorldTopoMap.

**Comment by bel28kent** [comment]
I have updated the tile layer code with Dr. Poudrier's chosen maps for the timeline map (timline-boilerplate.html) and the corpus map (corpus.html). Changes were only made on branch 26. I did not update the tile layer for the composers map as there doesn't seem to be an html file for that map.

**Comment by epoudrier** [comment]
The composers map is Jason’s.Sent from my iPhoneOn Apr 19, 2023, at 11:38 AM, Bryan Jacob Bell ***@***.***> wrote:﻿
I have updated the tile layer code with Dr. Poudrier's chosen maps for the timeline map (timline-boilerplate.html) and the corpus map (corpus.html). Changes were only made on branch 26. I did not update the tile layer for the composers map as there doesn't seem to be an html file for that map.
